sagemaker-pipelines/tabular/abalone_build_train_deploy/sagemaker-pipelines-preprocess-train-evaluate-batch-transform.ipynb

```diff
@@ -26,6 +26,7 @@
     "* Register model steps - A step that creates a model package resource in the Model Registry that can be used to create deployable models in Amazon SageMaker.\n",
```

Comment:
<p>1) suggested to put a comma in front of "Or Terminate" to break up that huge title</p><p>2) "the pipeline execution will be failed and terminated" -&gt; "the pipeline execution will fail and terminate"</p><p>3) remove the comma between "executed" and "if" in the last 2 bullet points</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Updated in the new commit.</p><p>As for 2), update the <code>the pipeline execution will fail and terminate</code> to be <code>the pipeline execution fails and terminates</code>, as its previous sentence uses the present tense</p>